main.go

```diff
@@ -80,6 +84,13 @@ func main() {
 	var options GlobalOptions
 	flagParser := flag.NewParser(&options, flag.PrintErrors)
 	flagParser.Usage = "-p <parser> -k <writekey> -f </path/to/logfile> -d <mydata> [optional arguments]"
+
+	// read the config file if present
+	options.ConfigFile = func(s string) error {
+		ini := flag.NewIniParser(flagParser)
+		return ini.ParseFile(s)
+	}
+
 	if extraArgs, err := flagParser.Parse(); err != nil || len(extraArgs) != 0 {
```

Comment:
I can't remember the decision wrt enforcing the `--config` option coming first on the command line.  Is that something we want to do?